app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,10 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = Post.where(user_id: params[@user.id])
+    @posts = @user.posts
   end
 
```

Comment:
![image](https://github.com/Cabraham1/Blog/assets/93039359/385db587-a2ee-42f9-886b-dda99ceeabf6)

**[OPTIONAL]**

- [ ] It would be nice if you make your readme summary more detailed to make your readme more descriptive and professional 👍